indigo_app/views/works.py

```diff
@@ -1318,3 +1318,44 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class WorkFormRepealsMadeView(WorkViewBase, TemplateView):
+    form_class = RepealMadeForm
+    template_name = 'indigo_api/_work_form_repeals_made_form.html'
+
+    def post(self, request, *args, **kwargs):
+        return self.get(request, *args, **kwargs)
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        RepealMadeBaseFormSet = formset_factory(RepealMadeForm, extra=0, can_delete=True)
```

Comment:
Minor: you set this up in three places -- the best place to do it is in forms.py, just beneath RepealMadeForm,.